extensions/db/src/main/java/org/mule/extension/db/internal/domain/executor/AbstractSingleQueryExecutor.java

```diff
@@ -32,7 +32,12 @@ public Object execute(DbConnection connection, Query query) throws SQLException
 
     prepareQuery(statement, query);
 
-    return doExecuteQuery(connection, statement, query);
+    try {
+    	return doExecuteQuery(connection, statement, query);
+    } catch (SQLException e) {
+    	LOGGER.error("Error executing "+query.getQueryTemplate().getSqlText(), e);
```

Comment:
I was thinking about this, and not convinced that this must be parameterized. So skip that until someone complains :)
However, there a couple of things that should be addressed:
1) This change is covering only the single query case, but it should cover the bulk update too.
2) Makes sense to throw a new exception instead of logging it here? (just to avoid logging the same exception twice)
3) The sqlText won't be helpful in all the cases, as it contains placeholders for parameterized queries. You can use the QueryLogger to properly log query parameters too